### ISSUE TITLE
Download URL not working again

### ISSUE THREAD

**Issue by andyy53** [fyu/lsun#20]
the LSUN download URL seems to have stopped working: 
http://lsun.cs.princeton.edu/htbin/list.cgi?tag=bedroom
http://lsun.cs.princeton.edu/challenge/2015/roomlayout/data/

**Comment by ricshaw** [comment]
@fyu Can you provide a link to the room layout challenge data for RoomNet?

**Comment by fyu** [comment]
Indeed, the downloading server for the main LSUN data is moved to dl.yf.io. The original Princeton server has expired. Please contact Yinda Zhang https://www.zhangyinda.com/ for room layout dataset.
